docs/dasharo-tools-suite/releases.md

```diff
@@ -13,64 +13,95 @@ For details about our release process please read
 
 [newsletter]: https://newsletter.3mdeb.com/subscription/ttzqCq9fy
 
-## v1.1.0
+## v1.1.1 - 2023-02-20
+
+### Images
+
+* [USB bootable DTS v1.1.1 image](https://3mdeb.com/open-source-firmware/DTS/v1.1.1/dts-base-image-v1.1.1.wic.gz)
```

Comment:
Could we point the release binaries in the same manner as Dasharo release binaries? I'm referring to the formatting method (buttons, not bullets).

Reply:
@sulewskiprzemyslaw done, https://github.com/Dasharo/docs/pull/369/commits/1da7504b0ca37c218e721909b5a683068cc19ee4#diff-dcdc85616220542189f6c2c350d6a3d92405bdee1142a24b3c56bf3484994b07R20 please resolve comment if ok